src/identity.js

```diff
@@ -592,8 +594,10 @@ export class Identity extends EventEmitter {
 
     /**
      * @async
-     * @summary In Schibsted account, there are two ways of identifying a user; the `userId` and the
-     * `uuid`. There are reasons for them both existing. The `userId` is a numeric identifier, but
+     * @summary
+     * In Schibsted account, there are multiple ways of identifying a user; the `userId`,
+     * `uuid` and `externalId` (see {@link Identity#getExternalId}). There are reasons for them all
```

Comment:
but externalId doesn't identify a user right? It identifies a user-merchant pair, so this might be a bit misleading, right ❓

Reply:
Fair point, will update!